rest-service/manager_rest/resource_manager.py

```diff
@@ -1398,6 +1398,20 @@ def get_deployment_parents_from_labels(labels):
                 parents.append(value)
         return parents
 
+    def get_deployment_object_types_from_labels(self, deployment, labels):
+        new_labels = self.get_labels_to_create(deployment, labels)
+        created_types = set()
+        delete_types = set()
+        for key, value in new_labels:
+            if key == 'csys-obj-type' and value:
+                created_types.add(value.lower())
+
+        new_labels_set = set(labels)
+        for label in deployment.labels:
+            if (label.key, label.value) not in new_labels_set:
+                delete_types.add(label.value.lower())
```

Comment:
Plus, this is the exact same code from https://github.com/cloudify-cosmo/cloudify-manager/blob/master/rest-service/manager_rest/resource_manager.py#L2316. 
So why not putting it in a different function, let's say `labels_to_delete`, and then run on the labels there and check if their key equals `csys-obj-type`?

Reply:
This way cannot make it generic for other places need to call labels_to_delete
```
        new_labels_set = set(labels)
        for label in deployment.labels:
            if label.key == 'csys-obj-type':
                if (label.key, label.value) not in new_labels_set:
                    delete_types.add(label.value.lower())
 ```

Reply:
This get updated now, ignore my last comment